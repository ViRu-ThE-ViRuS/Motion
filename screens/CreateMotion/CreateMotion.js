import React, { Component } from 'react';
import {
    View,
    Text,
    StyleSheet,
} from 'react-native';

export default class CreateMotion extends Component {
    render() {
        return (
          <View style={styles.container}>
            <Text>Hello</Text>
          </View>
        );
    }
}

const styles = StyleSheet.create({
    container: {

    },
});
