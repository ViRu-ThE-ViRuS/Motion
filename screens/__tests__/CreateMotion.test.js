import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import CreateMotion from './../CreateMotion/CreateMotion';

describe('Login screen tests', () => {
    let tree;
    let mockNavigation;
    let wrapper;

    beforeEach(() => {
        global.fetch = jest.fn((props) => ({
            text: jest.fn(),
        }));

        mockNavigation = {
            navigate: jest.fn((route) => { }),
        };

        wrapper = shallow(<CreateMotion navigation={mockNavigation} />);

        tree = renderer.create(
          <CreateMotion navigation={mockNavigation} />
          );
    });

    it('renders correctly', () => {
        expect(tree).toMatchSnapshot();
    });
});
