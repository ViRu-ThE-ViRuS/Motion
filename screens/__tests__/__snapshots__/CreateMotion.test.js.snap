// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login screen tests renders correctly 1`] = `
<View
  style={Object {}}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello
  </Text>
</View>
`;
